packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -366,7 +367,16 @@ export const actionCreators: ActionCreators = {
         getDevWorkspaceClient().checkForDevWorkspaceError(startingWorkspace);
       } catch (e) {
         // Skip unauthorised errors. The page is redirecting to an SCM authentication page.
-        if (common.helpers.errors.includesAxiosResponse(e) && isOAuthResponse(e.response.data)) {
+        if (
+          common.helpers.errors.includesAxiosResponse(e) &&
+          isOAuthResponse(
+            (
+              e as {
+                response: AxiosResponse;
+              }
+            ).response.data,
+          )
+        ) {
```

Comment:
This seems to me to be unnecessary changes

```suggestion
        if (common.helpers.errors.includesAxiosResponse(e) && isOAuthResponse(e.response.data)) {
```